commons-business/src/main/java/com/appjars/saturn/service/DeletionService.java

```diff
@@ -20,15 +20,20 @@
 package com.appjars.saturn.service;
 
 /**
- * A special kind of service that allows entities deletion
- * 
+ * Specifies the contract of a generic deletion service.
+ *
+ * @param <T> the type of entity (e.g., {@code User})
+ *
  * @author mlopez
  *
- * @param <T>
- * @param <K>
  */
 public interface DeletionService<T> {
 
-	void delete(T entity);
+    /**
+     * This method is responsible for deleting a given entity from the system.
```

Comment:
It seems that the conjugation is inconsistent between methods. Perhaps a better description would be `Deletes a given entity from the system.`